src/index.js

```diff
@@ -0,0 +1,204 @@
+import './styles/styles.css';
+
+const refresh = document.querySelector('.uil-sync');
+const input = document.querySelector('input');
+const todosParent = document.querySelector('.todos-container');
+const clearCompletedTaskBtn = document.querySelector('button');
+
+// Object
+class MyObject {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const myArray = [];
+
+// editTodo
+const editTodo = (todoContainer, todo) => {
+  const editInput = document.createElement('input');
+  editInput.type = 'text';
+  editInput.className = 'editInput';
+  editInput.value = todo.textContent;
+  todoContainer.replaceChild(editInput, todo);
+  editInput.addEventListener('keypress', (event) => {
+    if (event.key === 'Enter') {
+      const todoContainers = document.querySelectorAll('.todoContainer');
+      const localData = JSON.parse(localStorage.getItem('list'));
+
+      for (let i = 0; i < todoContainers.length; i += 1) {
+        if (todoContainers[i].classList.contains('completedTask')) {
+          localData[i].description = editInput.value;
+          localStorage.setItem('list', JSON.stringify(localData));
+        }
+      }
+      editInput.parentElement.classList.remove('completedTask');
+      todoContainer.replaceChild(todo, editInput);
+      todo.textContent = editInput.value;
+    }
+  });
+
+  // Delete Tasks
+  const removeIcons = document.querySelectorAll('.uil-trash-alt');
+  removeIcons.forEach((task) => {
+    task.addEventListener('click', () => {
+      // eslint-disable-next-line no-use-before-define
+      deleteTask(task.parentElement);
+    });
+  });
+};
+
+// Function to Delete Tasks
+const deleteTask = (todo) => {
+  // Delete task from browser
+  todosParent.removeChild(todo);
+
+  // Delete from localStorage
+  let count = 0;
+  const localData = JSON.parse(localStorage.getItem('list'));
+  const data = Array.from(localData).filter((task) => task.complete === false);
+  // eslint-disable-next-line no-return-assign, no-multi-assign
+  data.map((task) => task.index = count += 1);
+  localStorage.setItem('list', JSON.stringify(data));
+};
+// Update Local Storage
+const sendToLocalAgain = () => {
+  const localData = JSON.parse(localStorage.getItem('list'));
```

Comment:
- [ ] From my observation, **a task can only be deleted when it is marked as completed, and great job for that 👏 . It will be better if you could also add the functionality to delete a task even when it is NOT marked as completed** 😉